### ISSUE TITLE
[a11y] Add a note in Form validation

### ISSUE THREAD

**Issue by Aniort** [Orange-OpenSource/Orange-Boosted-Bootstrap#3052]
### Prerequisites

- [x] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

In https://ouds-main--boosted.netlify.app/docs/0.5/forms/validation/, I thinkk we must add a note:

- Due to constraints in how CSS works, we cannot (at present) apply styles to a <label> that comes before a form control in the DOM without the help of custom JavaScript.
- All modern browsers support the [constraint validation API](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#the-constraint-validation-api), a series of JavaScript methods for validating form controls. **But, not all couples, user agent/screen reader, support such an API feature, so do not use it without an accessibility fallback. **
- Feedback messages may utilize the [browser defaults](https://ouds-main--boosted.netlify.app/docs/0.5/forms/validation/#browser-defaults) (different for each browser, and unstylable via CSS) or our custom feedback styles with additional HTML and CSS.

### Motivation and context

https://a11ysupport.io/tests/tech__aria__aria-errormessage-with-aria-invalid-true#assertion-aria-aria-errormessage_attribute-convey_pertinent
